.github/workflows/assign_new_issue_to_project

```diff
@@ -0,0 +1,16 @@
+name: Auto Assign Issues to Projects
+
+on: issues
```

Comment:
We should filter out the PRs, if it doesn't already. I think they are considered issues by github.

Reply:
Can't find reference to that in the docs
https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#issues

Let's just try and see if it works as expected